app/controllers/users_controller.rb

```diff
@@ -61,11 +62,17 @@ def update
 
   private
 
+  def managers_only_create_editors!
```

Comment:
Would it be worth ticketing future work for how to move this into the ability, e.g.  `can :update, User, role: EDITOR`, for a subsequent PR, or is that kind of thing not possible?